src/components/Menu/MenuItem/MenuItem.tsx

```diff
@@ -204,7 +220,25 @@ const MenuItem: VibeComponent<MenuItemProps> & {
     const renderSubMenuIconIfNeeded = () => {
       if (!hasChildren) return null;
 
-      return (
+      return isSplitButton ? (
+        <div className={styles.subMenuIconWrapper}>
+          <Divider direction={DirectionType.VERTICAL} className={styles.divider} />
+          <div
+            onMouseEnter={() => setIsMouseOverSplitButton(true)}
+            onMouseLeave={() => {
+              setIsMouseOverSplitButton(false);
+            }}
+          >
+            <IconButton
+              icon={DropdownChevronRight}
```

Comment:
add `tabIndex={-1}` also, otherwise u'd be able to focus the button with the Tab key - we talked about it